src/wp-activate.php

```diff
@@ -128,7 +128,7 @@ function wpmu_activate_stylesheet() {
 	<?php if ( ! $key ) { ?>
 
 		<h2><?php _e( 'Activation Key Required' ); ?></h2>
-		<form name="activateform" id="activateform" method="post" action="<?php echo network_site_url( $blog_details->path . 'wp-activate.php' ); ?>">
+		<form name="activateform" id="activateform" method="post" action="<?php echo esc_url( network_site_url( $blog_details->path . 'wp-activate.php' ) ); ?>">
```

Comment:
There are two other instances in this file of `network_site_url()` that aren't wrapped in `esc_url()`, as well as some instances in other files where it's being called for output (or in assignment to a variable for output) but isn't escaped